### ISSUE TITLE
Streamie.org not working?

### ISSUE THREAD

**Issue by EugeneLiang** [cramforce/streamie#45]
Hi, i was trying out streamie.org and i got an error when i try to authenticate your app:
error: "error :{"statusCode":401,"data":"Failed to validate oauth signature and token"}"

Am i doing anything wrong?

Best.

**Comment by badboy** [comment]
No, streamie is just broken for some days now and cramforce is on his honeymoon.

But he is working on it according to his latest tweet: https://twitter.com/#!/cramforce/status/99123502308786176
